### ISSUE TITLE
Ingest ArGIS collection into VEDA dev catalog

### ISSUE THREAD

**Issue by smohiudd** [NASA-IMPACT/veda-backend#359]

**Comment by smohiudd** [comment]
I used the [example notebook](https://github.com/NASA-IMPACT/pyarc2stac/blob/main/examples/Convert_WMS_To_Collection_STAC.ipynb) in pyarc2stac to create a collection `ARCGIS_POWER_901_MONTHLY_RADIATION_UTC` in `dev`: https://dev.openveda.cloud/api/stac/collections/ARCGIS_POWER_901_MONTHLY_RADIATION_UTC

@j08lue can you let me know if the UI team can work with this collection?

cc: @slesaad, @amarouane-ABDELHAK, @anayeaye

**Comment by j08lue** [comment]
I see the collection has the `dashboard` metadata that VEDA UI depends on ✅ 

![image](https://github.com/NASA-IMPACT/veda-backend/assets/3404817/46a5e10d-3d5b-4d28-a257-3d8bee05d5c7)

It also has items but the list is empty.

![image](https://github.com/NASA-IMPACT/veda-backend/assets/3404817/768855e6-f96b-4c99-9551-f1f9cb16c530)

I guess we will identify this data source type somehow and then the UI uses the WMS link instead.

![image](https://github.com/NASA-IMPACT/veda-backend/assets/3404817/5fbadb5c-92f8-4347-97ce-aa2d3352e717)

We can perhaps adapt @slesaad's POC to try this out. I'll create a ticket for that.

- https://github.com/NASA-IMPACT/veda-ui/pull/887

**Comment by smohiudd** [comment]
Here are some notes for discussion and differences between Proxy and direct ingestion.

| Using Arc2Stac Proxy                                                                                                                    | Directly Ingest Collections                                                                                        |
| --------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
| Collections served up by proxy are always synced with ArcGIS. Summaries are always synced.<br><br>                                      | Collection may need to be re-ingested depending on the update frequency of the collection so summaries are in sync |
| To avoid high request hits to ArcGIS servers, we can employ a cacheing strategy for collections.                                        | Cacheing strategy not needed since the collections will be included in our STAC Catalog<br>                        |
| More infrastructure to deploy and manage using Arc2stac.                                                                                | Simpler infrastructure strategy since we are using the existing STAC catalog and database                          |
| `/statistics` endpoint can retrieve timeseries data. This results in statistics data being served in a consistent way to the front end. | No timeseries capabilities unless we access the ArcGIS `getSamples`endpoint in the frontend.                       |

**Comment by j08lue** [comment]
Great overview! We also discused the case of using a proxy for `/statistics` (to harmonize syntax), but not for `/collections`, right?

**Comment by smohiudd** [comment]
Yes I believe that was an option as well: ingest collections to STAC catalog along with a proxy just for statistics

**Comment by smohiudd** [comment]
Notes form the meeting on May 13 between VEDA Data Services and UI teams:
- A hybrid of the Arc2Stac Proxy and Collection ingestion is the preferred option going forward
- for consistency, scalability and future data integrations (i.e CMR) the preference would be to use a proxy for **statistics only.** This proxy would be it's own service and in the future may be a way to serve statistics with a specified schema from other sources
- ArcGIS collections will be directly ingested into the STAC catalog including external links for tiling endpoints such as WMS or WMTS. 
- Ingesting ArcGIS collections are an opportunity to explore and implement other meta data or extensions in collections
	- We will need a way to identify these unique collections in the catalog
	- Offload meta data from veda-config mdx files
	- include titiler endpoints
